src/clientapi/index.js

```diff
@@ -129,7 +129,8 @@ exports.ClientAPI = Class(lib.PubSub, function () {
   var spritesheets;
   try {
     if (GLOBAL.CACHE) {
-      spritesheets = JSON.parse(GLOBAL.CACHE['spritesheets/map.json']);
+      var spriteSheetCache = GLOBAL.CACHE['spritesheets/map.json'] || GLOBAL.CACHE['spritesheets\\map.json'];
```

Comment:
My original attempt at a fix was to edit `native-build.js` so I could catch the path and modify it on the way out of the server (line 25 is where the cache get's stringified) but doing a blind replace on the string seemed way riskier.